js/base/Exchange.js

```diff
@@ -1209,7 +1209,7 @@ module.exports = class Exchange {
             // the fee is always in the currency you get
             cost = amountString;
             if (side === 'sell') {
-                cost = priceString;
+                cost = Precise.stringMul (cost, priceString);
```

Comment:
@carlosmiei whenever you can to revise this PR also fixed the bug imposed by other PR, and quite critical update because of trading miscalculations for users.